add label delimiter

ueqt opened the pull request:
adding "\n" token as default option to separate long labels into different lines for Radar.
ref #229 
ref #228

etimberg commented:
Would this be useful to all chart types or only radar charts?

ueqt commented:
only for radar charts.

ckihneman commented:
Whats the word on this? Looking to push this into core? It would solve #608 as well.

etimberg commented:
I think this is mostly safe though it needs 2 small changes:
1. Remove `Chart.min.js` and `Chart.js` per the contributing guidelines
2. Account for the fact that the scaleFontSize is not always 12px
```javascript
for(var l=0;l<sepLabels.length;l++) {
    ctx.fillText(sepLabels[l], pointLabelPosition.x, pointLabelPosition.y + (l*12));
}
```

@nnnick @fulldecent @jakesyl @tannerlinsley @derekperkins @fntneves any thoughts?

ueqt commented:
Hi @etimberg ,

I've followed your suggestion, and changed code
1. Removed Chart.min.js and Chart.js
2. Removed magic number like 12px instead of scaleFontSize
3. Automatically increase the height based on multiple line height
4. Tooltip also support multiple lines

uklawitter commented:
:+1:

tannerlinsley commented:
We've recently announced a feature complete status for v1.x and will only be accepting bug fixes for v1.x. Be sure to take a look at v2.0-dev and it's new features that are either finished or currently under development.  If you see your feature is still not implemented, feel free to submit another PR to the [v2.0-dev](https://github.com/nnnick/Chart.js/tree/v2.0-dev) (or later) branch for your feature.
